modules/3d-tiles/src/tileset/helpers/bounding-volume.js

```diff
@@ -43,22 +42,14 @@ export function createBoundingVolume(boundingVolumeHeader, transform, result) {
       degrees((north + south) / 2),
       (minHeight + maxHeight) / 2
     );
+    result = result || {};
     Object.assign(result, boundingVolumeHeader, {center});
 
     return result;
     // return createRegion(boundingVolumeHeader.region, transform, this._initialTransform, result);
   }
   if (boundingVolumeHeader.sphere) {
-    // The first three elements define the x, y, and z values for the center of the sphere in a right-handed 3-axis (x, y, z)
```

Comment:
As mentioned, I do think the code to extract an initial suggested "view state" should be somewhere in the tileset rather than in the app (there are three type of bounding boxes, the app should not need to deal with that...) . 

Perhaps add a separate method `_getCartographicCenterAndZoom`?